frontera/contrib/backends/remote/messagebus.py

```diff
@@ -38,13 +38,13 @@ def frontier_stop(self):
         self.spider_log_producer.flush()
```

Comment:
Is this class tested somewhere? It's minor, but I don't see any code testing it.